src/config/configAggregator.ts

```diff
@@ -165,13 +166,22 @@ export class ConfigAggregator extends AsyncOptionalCreatable<ConfigAggregator.Op
 
   /**
    * Get a resolved config property.
+   * If you use a deprecated property, a warning will be emitted and it will attempt to resolve the new property's value
    *
    * **Throws** *{@link SfError}{ name: 'UnknownConfigKeyError' }* An attempt to get a property that's not supported.
    *
    * @param key The key of the property.
    */
   public getPropertyValue<T extends AnyJson>(key: string): Optional<T> {
-    if (this.getAllowedProperties().some((element) => key === element.key)) {
+    const match = this.getAllowedProperties().find((element) => key === element.key);
+    if (match?.deprecated && match.newKey) {
```

Comment:
Appears that the warning will only be emitted when a deprecated key has a new key. What about deprecation if a config value that does not have al alternative? Should we emit the warning when one is using a config key that does not have an alternative?

Reply:
we would not want to emit this particular warning/message.

I am not aware of any plans to deprecate and not replace a config